sur/tiebout.hoon

```diff
@@ -0,0 +1,62 @@
+::
+::::  /lib/tiebout/hoon
```

Comment:
Pretty sure filepaths in file headers is outdated style.

And as mentioned before, lots of the structures (cards, state) here really belong in the `/app` file instead, since they aren't shared to/used by other applications/libraries interfacing with Tiebout.